sconfig/shared/src/main/scala/org/ekrich/config/impl/SimpleConfigObject.scala

```diff
@@ -79,7 +81,7 @@ object SimpleConfigObject {
   }
 
   @SerialVersionUID(1L)
-  final private class RenderComparator
+  private class RenderComparator
```

Comment:
I'm not sure where this guarantee is used.
What rule does my new comparator have to obey?